README.md

```diff
@@ -23,13 +23,13 @@ A Descope `Project ID` is required to initialize the SDK. Find it on the
 [project page in the Descope Console](https://app.descope.com/settings/project).
 
 ```go
-import "github.com/descope/go-sdk/descope"
+import "github.com/descope/go-sdk/descope/client"
 
 // Initialized after setting the DESCOPE_PROJECT_ID env var
-descopeClient := descope.NewDescopeClient()
+descopeClient := client.NewDescopeClient()
```

Comment:
following [golang style guide](https://google.github.io/styleguide/go/decisions#repetition) , i think a better name will be `client.New()` (and `NewWithConfig` for the bellow)